### DISCUSSION TITLE
“Button:pointerover” is not working

### DISCUSSION THREAD

**Discussion by zy1075984** [AvaloniaUI/Avalonia#11942] [Q&A] [answered]
I write the following code, but after running the mouse moves to the button has no effect, may I ask why this may be?
<UserControl>
.....      
  <Button  >
     <Button.Styles>
        <Style Selector="Button:pointerover">
          <Setter Property="Background" Value="Red"/>
        </Style>
      </Button.Styles>
  </Button>
....

**Comment by zy1075984** [comment]
<Button.Styles>
            <Style Selector="Button:pointerover">
              <Setter Property="Background" Value="Red"/>
            </Style>
          </Button.Styles>

**Comment by danielmayost** [comment, marked as answer]
It's should be:
```
<Button.Styles>
  <Style Selector="Button:pointerover /template/ ContentPresenter">
    <Setter Property="Background" Value="Red"/>
  </Style>
</Button.Styles>
```

**Comment by timunie** [comment]
@zy1075984 please do us a favor: If you think you found a bug, open an issue using the provided templates. If you have just a question, please use Discussions or join us on [telegram](https://t.me/Avalonia). Also please always let us know the version you use. 

I'm moving this over to discussions now. 

Thank you for your understanding 🙏
